### ISSUE TITLE
[Package]:Please Add this things to python wiki

### ISSUE THREAD

**Issue by mablue** [termux/termux-packages#15048]
### Why do you want to add this package to Termux?
When a python programmer wants to use termux as his main os many problems come to dissuade him. What do we do for it? there is many python programmers now (after some updates to important packages of python) that never never want to come back to termux and them going to use their laptops and google colab instade of termux. we need an strong help to fix this problems. we need a good wiki to help python programmers. but its not the best way. we should update termux to be like a normal linux system or make a packages to help people about installiing important python packages. There are some sorts of commands that I always use for installing things on different phones, but I still have many problems too. And it's very hard to understand what these different phones need as a dependency for installing and using different python packages.(ccxt, pandas, cryptography, ta-lib, etc...)
```
# ~/.bashrc modifications for python dev

PS1='\[\e[0;32m\]\W\[\e[0m\] \[\e[0;97m\]\$\[\e[0m\] '

export CFLAGS="-Wno-deprecated-declarations -Wno-unreachable-code"
export MATHLIB="m"
export RUSTFLAGS=" -C lto=no"
export CARGO_BUILD_TARGET="$(rustc -vV | sed -n 's|host: ||p')"

cd storage/shared/
```
Intro to install important python packages
```
pkg install ta-lib
pip install ta-lib
pip install 
pip install --no-use-pep517 pandas
```

### Home page URL

https://wiki.termux.com/wiki/Python

### Source code URL

https://wiki.termux.com/wiki/Python

### Packaging policy acknowledgement

- [X] The requested package complies with [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy).

### Additional information

```

PS1='\[\e[0;32m\]\W\[\e[0m\] \[\e[0;97m\]\$\[\e[0m\] '

export CFLAGS="-Wno-deprecated-declarations -Wno-unreachable-code"
export MATHLIB="m"
export RUSTFLAGS=" -C lto=no"
export CARGO_BUILD_TARGET="$(rustc -vV | sed -n 's|host: ||p')"


cd storage/shared/\!\!SMALL/


pkg install python-cryptography python-numpy rust
pip install Cython
pip install --no-use-pep517 pandas
pip install cryptography 

pkg install rust

############################
export CARGO_BUILD_TARGET=armv7-linux-androideabi && pip install cryptography and export CARGO_BUILD_TARGET=aarch64-linux-android && pip install cryptography 

uname -m
#for arm7
export CARGO_BUILD_TARGET=armv7-linux-androideabi && pip install cryptography

#for arm8
export CARGO_BUILD_TARGET=aarch64-linux-android && pip install cryptography
############################

pip install pynacl
pkg install libsodium
pkg install binutils
pip install pyproject-toml
pip install wheel
pip install lxml
pkg install libxml2
pkg install libxslt
pip install cython


export CFLAGS="-Wno-deprecated-declarations -Wno-unreachable-code"
pip install pandas

pkg install python-tkinter

pkg install libjpeg-turbo libpng

#for 64bit
export LDFLAGS="-L/system/lib64"

pkg install pillow 
MATHLIB=m pip install numpy


```

**Comment by xtkoba** [comment]
Yes, PRs are welcome to fix these issues. Thanks in advance for your contributions.

**Comment by mablue** [comment]
Sorry if I use unfamiliar form of issue composing but it's the words of someone that use many phones to run python scripts on termux and I really tierd 😩. 
We can make the way easy to important packages...
also I think many people just installing the termux and never know there is a python wiki that them can read befoure try to installing things with pip!
for example I try 2days to install ta-lib with normal methods:
https://www.freqtrade.io/en/2020.3/installation/#ta-lib-manual-installation
but I think you know what Happend at the end!:))
Cuz of that It can be helpfull that we notify python programmers inside app that there is a termux wiki for python.
thanks for support

**Comment by 2096779623** [comment]
Ping @Neo-Oli .

**Comment by SunPodder** [comment]
Actually you need to be an experienced Linux User to solve these problems yourself. It doesn't have anything to do with termux. Moreover, termux doesn't have a gui which makes it more difficult for new users to understand what's actually happening. Compiling native code is kinda painful for beginners. I suggest to avoid them as much as you can. You can use [TUR](https://github.com/termux-user-repository/tur) instead to download those packages (pre-compiled). If something isn't available there, you can always open a package request for it.

**Comment by mablue** [comment]
yes but never a normal linux user even in colab have not this like problems.
we have here python and etc users that just want to have fun with linux and learn it. Mobile user is never can be like a pc user advance and he/she just want to do thigs more easy and fast. also we dont have download problem please append this small things natively inside termux to make a good env to users.  you know many people can be tierd when them can not install little pip packages esely like each other linux distro.

**Comment by xtkoba** [comment]
Kind of horrible grammar... I strongly recommend that you learn how to write in proper English first. You could not contribute to our documentation without that.

**Comment by ChiefMikeK** [comment]
## FYI 
[Draft namespace - MediaWiki](https://m.mediawiki.org/wiki/Draft_namespace ) **if implemented** could allow you to create an article even if you are not proficient in the English language and it's grammatical syntax!

Others can then edit/improve upon your content before it is published to the public NAMESPACE 

Please don't down trodden non-english native/first language speaking humans 🌏😣

**Comment by mablue** [comment]
> Kind of horrible grammar... I strongly recommend that you learn how to write in proper English first. You could not contribute to our documentation without that.

edited sorry its cuz of that my father seen my mother in Malekan city near of Tabriz city in iran that all two just can speak in Turkish and Persian. I hate to my English grammer but I can't fix it cuz of my genes...

**Comment by mablue** [comment]
closed cuz of gene difference more than 99.5%
